### DISCUSSION TITLE
SLES15 SP4 OS support

### DISCUSSION THREAD

**Discussion by mahfou3q** [cocotb/cocotb#3759] [Q&A]
I am Tring to find out if SLES15 SP4 OS is supported with COCOTB and which release?
i am Tring with cocotb1.6 and found the SO library of COCOtb not working.  

Thanks,

**Comment by ktbarrett** [comment]
Always try the most recent version of cocotb. Your OS isn't officially supported, but it should work. Please add more details so we can help you. Python version, simulator, simulator version, and the full error message.

**Reply by mahfou3q** [reply]
i am using Questasim 2024.1 version and Python3.7.9 
now i am facing that the messages should printed fron COCOTB not printed and the only found in the transcript is sv display only  
same testbench working fine with normal linux 64 OS 
4,6d3
<       0.00ns INFO     Running tests with cocotb v1.6.0 from cocotb
<       0.00ns INFO     Found test issue_120.issue_120_scheduling
<       0.00ns INFO     running issue_120_scheduling (1/1)
9,13d5
<      60.00ns INFO     issue_120_scheduling passed
<      60.00ns INFO     ****************************************************************************************
<                       ** issue_120.issue_120_scheduling   PASS
<                       ** TESTS=1 PASS

**Reply by ktbarrett** [reply]
Have you tried with cocotb v1.8.1? You can upgrade using `pip install --upgrade cocotb`.
